vip-application/src/main/java/fr/insalyon/creatis/vip/application/client/bean/Task.java

```diff
@@ -34,15 +34,20 @@
 import com.google.gwt.user.client.rpc.IsSerializable;
 import fr.insalyon.creatis.vip.application.client.view.monitor.job.TaskStatus;
 
+import java.util.Date;
+
 /**
  *
  * @author Rafael Ferreira da Silva
  */
 public class Task implements IsSerializable {
 
     private String id;
+    private int invocationID;
+    private Date creationDate;
     private TaskStatus status;
     private int exitCode;
+    private String exitMessage;
```

Comment:
`exitMessage` should not be a field of Task as it can be derived from `exitCode`.
No need no include it in constructor or in a setter.
Only a getter is needed.